DOCS-2541: Move component APIs (#3504)
viamrobotics/docs: docs/components/power-sensor/_index.md

{{% /tab %}}
{{< /tabs >}}

## API

The [power sensor API](/appendix/apis/components/power-sensor/) supports the following methods:

{{< readfile "/static/include/components/apis/generated/power_sensor-table.md" >}}

## Troubleshooting

You can find additional assistance in the [Troubleshooting section](/appendix/troubleshooting/).

You can also ask questions on the [Viam Community Slack](https://join.slack.com/t/viamrobotics/shared_invite/zt-1f5xf1qk5-TECJc1MIY1MW0d6ZCg~Wnw) and we will be happy to help.

## Next steps

For general configuration and development info, see:

{{< cards >}}
{{% card link="/how-tos/configure/" noimage="true" %}}
{{% card link="/how-tos/develop-app/" noimage="true" %}}
{{% card link="/how-tos/collect-sensor-data/" noimage="true" %}}
{{< /cards >}}

> To capture data from the power sensor or use it for motion, see the [data management service](/services/data/).

Reviewer: I don't know if this makes sense, do we use power sensors for motion?
```suggestion
To capture data from the power sensor and sync it in the cloud, see the [data management service](/services/data/).
```
